bindings/SwiftCore.xml

```diff
@@ -211,5 +211,6 @@
         <operator name="&amp;=" operatorKind="Infix" precedenceGroup="AssignmentPrecedence" moduleName="Swift" />
         <operator name="|=" operatorKind="Infix" precedenceGroup="AssignmentPrecedence" moduleName="Swift" />
         <operator name="^=" operatorKind="Infix" precedenceGroup="AssignmentPrecedence" moduleName="Swift" />
+	<typealias name="Foundation.NSRect" type="CoreGraphics.CGRect" />
```

Comment:
Legit for 64bits :)

/Applications/Xcode_12.5.0-rc.app//Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSGeometry.h
```
#if __LP64__ || TARGET_OS_IPHONE || NS_BUILD_32_LIKE_64
```

Reply:
Kill me now.